apps/console/src/features/applications/constants/application-management.ts

```diff
@@ -348,9 +348,7 @@ export class ApplicationManagementConstants {
     // Second factor authenticators.
     public static readonly SECOND_FACTOR_AUTHENTICATORS: string[] = [
         IdentityProviderManagementConstants.TOTP_AUTHENTICATOR,
-        IdentityProviderManagementConstants.TOTP_AUTHENTICATOR_ID,
-        IdentityProviderManagementConstants.SMS_OTP_AUTHENTICATOR,
```

Comment:
SMS OTP should've been removed as a second factor authenticator.

Like we've done for Email OTP: https://github.com/wso2/identity-apps/pull/3808/files#diff-5a4aee5fb6bc30d80cd0c7747bc71f559c89890d856edb398b1025322cbe62b5L345